packages/strapi-admin/admin/src/components/FormBloc/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Padded, Text } from '@buffetjs/core';
+import { LoadingIndicator, Row } from 'strapi-helper-plugin';
+import PropTypes from 'prop-types';
+import BaselineAlignement from '../BaselineAlignement';
+import Bloc from '../Bloc';
+
+const FormBloc = ({ children, isLoading, title }) => (
+  <Bloc>
+    <BaselineAlignement top size={title ? '18px' : '22px'} />
+    <Padded left right size="sm">
+      {isLoading ? (
+        <>
+          <LoadingIndicator />
+          <BaselineAlignement bottom size="22px" />
+        </>
+      ) : (
+        <>
+          {title && (
+            <>
+              <Padded left right size="xs">
```

Comment:
@HichamELBSI with the current `<Padded />` we start to add a lot of nested divs. What do you think about adding some props so we can use it as follows (with better props name):

```
<Padded leftRight="xs" toBottom="sm" />
<Padded all="xs" />
// And the current one
<Padded left right size="xs">
```